R/check-class.R

```diff
@@ -247,3 +280,56 @@ first_line <- function(x) {
   l <- strsplit(x, "\n", fixed = TRUE)
   vapply(l, "[[", "", 1)
 }
+
+#' Retrieve the result of R-hub checks
+#'
+#' @param ids One of the following:
+#'  - A single R-hub check id. It may also be the beginning of the hash
+#'    of the id (i.e. the part after the last dash), if that check was
```

Comment:
this part is unclear to me, if it is the beginning why is the part after the last dash?

Reply:
explained better in `?"rhub-ids"`